pkg/api/vlabs/types.go

```diff
@@ -226,32 +254,6 @@ type KeyVaultCertificate struct {
 	CertificateStore string `json:"certificateStore,omitempty"`
 }
 
-// OrchestratorType defines orchestrators supported by ACS
-type OrchestratorType string
```

Comment:
3. OrchestratorVersion is not going to be a field taking input from api model. It will be a read-only field. I am going to do this change, not in this PR yet.